SU2_PY/direct_differentiation.py

```diff
@@ -48,15 +48,19 @@ def main():
     parser.add_option("-n", "--partitions", dest="partitions", default=1,
                       help="number of PARTITIONS", metavar="PARTITIONS")
     parser.add_option("-q", "--quiet",      dest="quiet",      default='False',
-                      help="output QUIET to log files", metavar="QUIET")    
-    
+                      help="output QUIET to log files", metavar="QUIET")
+    parser.add_option("-z", "--zones", dest="nzones", default="1",
+                      help="Number of Zones", metavar="ZONES")
```

Comment:
We have this because the python scripts also need to know the number of zones in order to have to correct file names.